doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -1,17 +1,119 @@
 cells:
 
 - markdown: |
-    # Load/Dump Arrays And Sessions From/To Files
+    # Load And Dump Arrays, Sessions, Axes And Groups
 
 
 - markdown: |
-    Import the LArray library:
+    LArray provides methods and functions to load and dump LArray, Session, Axis Group objects to several formats such as Excel, CSV and HDF5.
 
 
 - code: |
+    # first of all, import the LArray library
     from larray import *
 
-  id: 1
+  id: 0
+  metadata:
+    nbsphinx: hidden
+
+- markdown: |
+    ## Load and Dump Arrays
+    
+    ### Excel
+    
+    #### Loading an Array from an Excel Sheet
+    
+    ##### Basic Options
+    
+    ##### Specifying the Number of Axes
+    
+    ##### NA and Missing Data Handling
+    
+    ##### Sorting Axes
+    
+    ##### Specifying Wide VS Narrow format
+    
+    ##### Specifying Position in Sheet (open_excel only)
+    
+    #### Dumping an Array to an Excel Sheet
```

Comment:
unsure where to place this but we need to show dumping and reading with header or without headers, including reconstructing an array with axes labels in arbitrary positions